src/models/subgraph-entities/v1/distribute-to-payout-mod-event.ts

```diff
@@ -22,27 +21,15 @@ export interface DistributeToPayoutModEvent extends BaseEventEntity {
   tapEvent: string
 }
 
-export type DistributeToPayoutModEventJson = Partial<
-  Record<Exclude<keyof DistributeToPayoutModEvent, 'project'>, string> & {
-    project: ProjectJson
-  } & BaseEventEntityJson
->
-
 export const parseDistributeToPayoutModEvent = (
-  j: DistributeToPayoutModEventJson,
+  j: Json<DistributeToPayoutModEvent>,
 ): Partial<DistributeToPayoutModEvent> => ({
-  ...parseBaseEventEntityJson(j),
```

Comment:
these subgraph-entities/** files have mostly the same changes. lots of duplicate code removed using a few handy parsing utils